iidm/iidm-api/src/main/java/com/powsybl/iidm/network/RatioTapChangerAdderStepAdder.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package com.powsybl.iidm.network;
+
+/**
+ * @author Florent MILLOT {@literal <florent.millot at rte-france.com>}
+ */
+public interface RatioTapChangerAdderStepAdder extends RatioTapChangerStepAdder<RatioTapChangerAdderStepAdder, RatioTapChangerAdder> {
```

Comment:
This is exactly the type of comment I was thinking of.

Maybe it could be slightly rephrased in something like:
`Interface for classes responsible of building a single step when using {@link RatioTapChangerAdder}.`
since it is not the interface which builds the steps.

Reply:
Ok done